File: ListaPOOExercicios/Atividade2/Python/Carro.py
class Carro:
    def __init__(self, marca, modelo, ano, velocidade):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano
        self.velocidade = velocidade

    def detalhes_carro(self):
        print(f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}, Velocidade: {self.velocidade} KM/h")

    def acelerar(self, aceleracao):
        self.velocidade += aceleracao

    def frear(self, desaceleracao):
        self.velocidade -= desaceleracao


carro1 = Carro("Fiat", "Uno", "2020", 0)
carro2 = Carro("Volkswagen", "Gol", "2019", 0)
carro3 = Carro("Chevrolet", "Camaro", "2015", 0)
carro1.acelerar(10)
carro2.acelerar(21)
carro3.acelerar(30)
carro2.frear(10)
carro1.detalhes_carro()
carro2.detalhes_carro()
carro3.detalhes_carro()

File: ListaPOOExercicios/Atividade2/Lista1Atividade2-Java/src/Carro.java
public class Carro {
    public String marca;
    public String modelo;
    public String ano;
    public double velocidade;

    public Carro(String marca, String modelo, String ano, double velocidade){
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
        this.velocidade = velocidade;
    }

    public void detalhesCarro(){
        System.out.println("Marca: "+marca+
                        ", Modelo: "+modelo+
                        ", Ano: "+ano+
                        ", Velocidade: "+velocidade+"KM/h");
    }
    // Acelerar e Frear
    public void acelerar(double aceleracao){
        velocidade += aceleracao;
    }
    public void frear(double aceleracao){
        velocidade -= aceleracao;
    }


}
